homeassistant/components/lifx/strings.json

```
@@ -220,6 +220,36 @@
         }
       }
     },
+    "effect_sky": {
+      "name": "Sky effect",
+      "description": "Starts the firmware-based Sky effect on LIFX Ceiling.",
+      "fields": {
+        "speed": {
+          "name": "Speed",
+          "description": "How long the Sunrise and Sunset sky types will take to complete. For the Cloud sky type, it is the speed of the clouds across the device."
+        },
+        "sky_type": {
+          "name": "Sky type",
+          "description": "(Optional, default: 'Clouds') The style of sky that will be animated by the effect."
+        },
+        "cloud_saturation_min": {
+          "name": "Cloud saturation Minimum",
+          "description": "(Optional, default: 50) Minimum cloud saturation."
```

```suggestion
          "description": "Minimum cloud saturation."
```

The default will already be filled in the UI if no value is entered since its defined in `services.yaml`